cris-cookies.md

```diff
@@ -2,8 +2,6 @@
 
 *vegetarian, cookies*
 
-**equals __ cookies** // no idea, please tell me how much you get out of it
-
```

Comment:
[The yield paragraph must be completely bold.](https://recipemd.org/specification.html#recipe) Here you're putting stuff after the end of the bold section, which makes `recipemd` choke up, since it did recognize the previous paragraph as the tags. After the tags, the only valid things to follow are the yields or the divider to indicate the start of the ingredient list.

The yields is optional, so if you don't know it you don't have to specify it. Thus I just removed it :)